### ISSUE TITLE
Improve documentation and dataset annotations for predictions

### ISSUE THREAD

**Issue by tobylunt** [devdatalab/covid#22]
The `hospitals_dist.dta` file doesn't line up exactly with [the documentation](https://docs.google.com/spreadsheets/d/e/2PACX-1vQL3zu-_LMCV3gIjk1NOGWJVwDwcspl2lI7lvvRUuUgRN3I-X9QFe9BUhdr3dhROpTItTiiUxP_-kU9/pubhtml?urp=gmail_link&gxids=7757), and many of the variables are unlabeled, which makes things unclear.

Please label the data more clearly and fix the metadata so it accurately reflects the posted data.

**Comment by aditib738** [comment]
I spent some time looking into this - here is a note on all of this:

- there were two datasets `pc11r_hosp.dta` and `pc11u_hosp.dta` that were basically just collections of variables from the pc11 village and town directories that were used to create the final hospital datasets we put out, so these were intermediate. They also dont have metadata sheets that go with them. So, I've put them in `iec/health/hosp/` for now to not confuse external users. For hospitals data - they should go to `dropbox/covid/hospitals/pc_hospitals_[dist/subdist].dta`. The link to the documentation is also correct for these datasets.

- in the hospitals subfolder, there are three EC datasets: `ec_hosp_microdata`, `ec_hosp_dist` and `ec_hosp_tv`. The microdata isn't labelled, and seems to be another intermediate dataset - i hadn't worked with the EC hospital estimates though so I haven't moved them around. `ec_hosp_dist` and `ec_hosp_tv` are just district and town/village level public private hospital numbers, and workforce size in pub/priv hospitals. **One issue I picked on is** that the link to the metadata for EC hospitals datasets was wrong in our documentation table -- it was taking us to the metadata for `pc_hospitals_[subdist/dist]`.. maybe this is what confused the user @tobylunt ? I've made an edit in the readme to take us to the [correct link now](https://docs.google.com/spreadsheets/d/1h6G4vYL3lvy4Bi8DTY3pMT2-5aVWOBoxAm3plx4M7qQ/edit?usp=sharing).

- the metadata you had shared on the slack channel belongs to `dropbox/estimates/hospitals_dist.dta`. This dataset is different from the ^^ hospital datasets as these have some imputed variables. 

"Health system capacity is based on public hospital bed counts from DLHS4 (2014) and Population Census (2011), and private:public employment ratios from the Economic Census (2013). We predict private bed counts by assuming that the public:private hospital employment ratio in the Economic Census is the same as the public:private hospital bed ratio. Our public-private share estimates broadly match other sources of data at the state level, including the National Health Profile 2019."

This dataset has some un-labelled variables, I can go ahead and label them - I'll check with Paul if some variables are unclear since he had done the imputation using the EC hospitals dataset he prepared, the PC hospitals dataset I had prepared, and the DLHS number (I think) Sam had put together (in hindsight, what a mad rush it was to get the data out! :) )
